.drone.yml

```diff
@@ -128,6 +128,11 @@ build:
       - export VM2=$WORKER2
       - ./misc/drone-scripts/deploy-and-test-wrapper.sh runtests $GOVC_URL_6_5 $VM1 $VM2 $$BUILD_NUMBER < /dev/null
       - #
+      - echo "=> Build and run tests for windows vDVS plugin" `date` > /dev/null
```

Comment:
nit: `Build and run tests for windows vDVS plugin` -> `Building and running tests for the Windows vDVS plugin`